clientapi/auth/password.go

```diff
@@ -62,6 +70,101 @@ func (t *LoginTypePassword) Login(ctx context.Context, req interface{}) (*Login,
 			JSON: jsonerror.InvalidUsername(err.Error()),
 		}
 	}
+	if len(t.Config.LDAP.Host) > 0 {
+		addr := ""
+		if t.Config.LDAP.TLS {
+			addr = "ldaps://" + t.Config.LDAP.Host + ":" + t.Config.LDAP.Port
+		} else {
+			addr = "ldap://" + t.Config.LDAP.Host + ":" + t.Config.LDAP.Port
```

Comment:
It might be easier to just have `t.Config.LDAPAddr` which can then mux together TLS/host/port e.g `ldaps://localhost:8008`

Reply:
Good idea, it should be changed like this.